tests/testsDumper.cpp

```diff
@@ -476,6 +476,26 @@ TEST(StringDumperTest, StringControlChars) {
             Dumper::toString(slice));
 }
 
+TEST(StringDumperTest, SuppressControlChars) {
+  Builder b;
+  b.add(Value("Before\nAfter\r\t\v\x01\x02/"));
+
+  Options options;
+  options.escapeControl = false;
+  ASSERT_EQ(std::string("\"Before After     /\""),
+            Dumper::toString(b.slice(), &options));
+}
+
+TEST(StringDumperTest, EscapeControlChars) {
+  Builder b;
+  b.add(Value("Before\nAfter\r\t\v\x01\x02"));
```

Comment:
The `EscapeTable` in Dumper.cpp has these different values for escaping:
`\uxxxx`, `\b`, `\t`, `\n`, `\f`, `\r`, `\"`, `\/`, `\\`
These should all be tested here.